crates/api_models/src/admin.rs

```diff
@@ -621,39 +650,63 @@ pub struct MerchantConnectorUpdate {
     pub metadata: Option<pii::SecretSerdeValue>,
 
     /// contains the frm configs for the merchant connector
-    #[schema(example = json!([
-        {
-            "frm_enabled_pms" : ["card"],
-            "frm_enabled_pm_types" : ["credit"],
-            "frm_enabled_gateways" : ["stripe"],
-            "frm_action": "cancel_txn",
-            "frm_preferred_flow_type" : "pre"
-        }
-    ]))]
-    pub frm_configs: Option<FrmConfigs>,
+    #[schema(example = json!([{
```

Comment:
try moving the example as a CONST and use it here. The example is repeated in multiple places